dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/domain/Enrollment.java

```diff
@@ -81,12 +81,12 @@ public class Enrollment implements TrackerDto, Serializable {
 
   @Override
   public UID getUid() {
-    return UID.of(enrollment);
+    return enrollment;
   }
 
   @Override
   public String getStringUid() {
-    return enrollment;
+    return enrollment.getValue();
```

Comment:
just for me to understand, we still need this method because the database schema doesn't accept a UID, right?